tests/ipc.c

```diff
@@ -81,6 +81,10 @@ int main ()
     /*  Send something large enough to trigger overlapped I/O on Windows. */
     size = 10000;
     buf = malloc (size);
+    if (!buf){
```

Comment:
I'd normally *not* do this for a test case, since the panic will be sufficient to debug, and this isn't part of the library core.  But I'll leave it alone for now.